src/manufacturer/manufacturerLandingPage.component.tsx

```diff
@@ -237,21 +203,10 @@ function ManufacturerLandingPage() {
       )}
       {!manufacturerDataLoading ? (
         !manufacturerData && (
-          <Box
-            sx={{
-              width: '100%',
-              justifyContent: 'center',
-              marginTop: '8px',
-            }}
-          >
-            <Typography sx={{ fontWeight: 'bold', textAlign: 'center' }}>
-              No result found
-            </Typography>
-            <Typography sx={{ textAlign: 'center' }}>
-              This manufacturer doesn&#39;t exist. Please click the Home button
-              to navigate to the manufacturer table
-            </Typography>
-          </Box>
+          <ErrorPage
```

Comment:
Should this be in the router? Unless the error handling you showed me was in another PR.

Reply:
https://github.com/ral-facilities/inventory-management-system/pull/1228